pipelines/srp-pipeline.yml

```diff
@@ -0,0 +1,522 @@
+
+############## Resources ################
+resources:
+  #### Secure Release Pipeline resources
+  - name: srp-helper-test
+    type: registry-image
+    source:
+      repository: harbor.dhaka.cf-app.com/srp/srp-helper-task
+      username: ((phoenix-osspi.srp-harbor-username))
+      password: ((phoenix-osspi.srp-harbor-password))
+
+  - name: build-metadata
+    type: build-metadata
+
+  - name: metric-store-ci
+    type: git
+    source:
+      uri: git@github.com:cloudfoundry/metric-store-ci
+      private_key: ((phoenix-osspi.svcboteos_github_private_key))
+      branch: srp-scan
+
+  - name: metric-store-develop
+    type: git
+    source:
+      uri: git@github.com:cloudfoundry/metric-store-release
+      branch: master
+      private_key: ((phoenix-osspi.svcboteos_github_private_key))
+      ignore_paths:
+        - .final_builds
+        - releases
+      disable_ci_skip: false
+      clean_tags: true
+
+  - name: metric-store-version
+    type: semver
+    source:
+      driver: gcs
+      bucket: metric-store-ci-versions
+      key: test/metric-store-release-acceptance-tile
+      json_key: ((phoenix-osspi.gcp_service_account_key))
+      initial_version: 0.2.0-build.24
+
+
+############## Jobs ################
+jobs:
+  ##### Secure Release Pipeline jobs
+  - name: "collect-srp-source-provenance-metric-store"
+    serial: true
+    plan:
+      - in_parallel:
+          - get: srp-helper-test
+          - put: build-metadata
+          - get: metric-store-version
+          - get: metric-store-develop
+          - get: metric-store-ci
+      - task: "submit-metadata-for-metric-store"
+        image: srp-helper-test
+        config:
+          platform: linux
+          inputs:
+            - name: metric-store-ci
+            - name: metric-store-develop
+            - name: build-metadata
+            - name: metric-store-version
+          params:
+            SRP_CLIENT_ID: ((phoenix-osspi.srp-client-id))
+            SRP_CLIENT_SECRET: ((phoenix-osspi.srp-client-secret))
+            SRP_URL: https://srp.vmware.com
+            SRP_KEY: ((phoenix-osspi.svcboteos_github_private_key))
+          run:
+            path: /bin/bash
+            args:
+              - -c
+              - |
+                set -x
+                set -euo pipefail
+                mkdir ./provenance
+
+                VERSION=$(cat metric-store-version/version)
+                BUILD_NUMBER=$(cat build-metadata/build-name)
+                BUILD_PIPELINE_NAME=$(cat build-metadata/build-pipeline-name)
+                BUILD_JOB_NAME=$(cat build-metadata/build-job-name)
+                BUILD_ID=$(cat build-metadata/build-id)
+                OUTPUT_VERSION="1.0.0-${BUILD_ID}"
+                BUILD_PROJECT=metric-store-release
+                BUILD_BRANCH=$(cat metric-store-develop/.git/refs/heads/master)
+                ARTIFACTORY_AUTH_TOKEN=cmVmdGtuOjAxOjE3MjYyMjY1MDI6cWU2U1FlazVjNjVoNUllV2pWWU1BQ0NnaHJV
```

Comment:
Need to store token in the https://runway-vault.eng.vmware.com/ui/vault/secrets/runway_concourse/show/tobs-k8s-group/ then pass it as a` params:` like this:
params:
            SRP_CLIENT_ID: ((phoenix-osspi.srp-client-id))
            SRP_CLIENT_SECRET: ((phoenix-osspi.srp-client-secret))
            SRP_URL: https://srp.vmware.com
            SRP_KEY: ((phoenix-osspi.svcboteos_github_private_key))
            `ARTIFACTORY_AUTH_TOKEN: (phoenix-osspi.artifactory_token))`

Reply:
Done